themes/default/content/docs/guides/crosswalk/aws/api-gateway.md

```diff
@@ -291,549 +313,452 @@ specifying the `target` property on your route.
 
 An Integration Route is a route that will map an endpoint to a specified backend. The supported types are:
 
-* `aws`: This type of integration lets an API expose AWS service actions, such as invoking Amazon Lambda Functions,
+- `aws`: This type of integration lets an API expose AWS service actions, such as invoking Amazon Lambda Functions,
   Amazon DynamoDB, Amazon Simple Notification Service, or Amazon Simple Queue Service. You must set up the
   necessary data mappings between the HTTP and underlying AWS service requests/responses.
-* `aws_proxy`: This type of integration lets an API expose AWS service actions, much like `AWS`, and passes
+- `aws_proxy`: This type of integration lets an API expose AWS service actions, much like `AWS`, and passes
   the HTTP request information, including request headers, URL path variables, query string parameters, and
   applicable body, directly to the underlying AWS service actions.
-* `http`: This type of integration lets an API expose HTTP endpoints with custom integration requests and responses.
+- `http`: This type of integration lets an API expose HTTP endpoints with custom integration requests and responses.
   You must set up necessary data mappings between the HTTP and integration requests/responses.
-* `http_proxy`: This type of integration lets an API expose HTTP endpoints with a streamlined integration,
+- `http_proxy`: This type of integration lets an API expose HTTP endpoints with a streamlined integration,
   without needing to perform custom data mappings as with the `HTTP` integration type.
-* `mock`: This type of integration lets API Gateway return a response without sending a request further to
+- `mock`: This type of integration lets API Gateway return a response without sending a request further to
   the backend. This is useful for API testing without needing to configure any backend to service requests.
 
 The following example sets up an `http_proxy` integration type that simply passes requests/responses directly
 through to another endpoint, in this case `https://www.google.com`:
 
-```typescript
-import * as awsx from "@pulumi/awsx";
-
-// Define a GET endpoint that proxies HTTP requests to https://www.google.com.
-const api = new awsx.apigateway.API("example", {
-    routes: [{
-        path: "/google",
-        target: {
-            type: "http_proxy",
-            uri: "https://www.google.com",
-        },
-    }],
-});
-
-// Export the auto-generated AWS API Gateway base URL.
-export const url = api.url;
-```
-
-For more information AWS API Gateway Integrations, visit the
-[AWS documentation](https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-api-integration-types.html).
+{{< chooser language "typescript,python,go" / >}}
 
-### Defining an OpenAPI Specification for an Entire Endpoint
```

Comment:
Did we end up removing this section?  It's fairly useful to be able to highlight that this is possible, even if we might want to scale back the depth of this content a little.

Reply:
Ahh - I see it was moved to the end.